packages/polywrap-client/tests/test_sha3.py

```diff
@@ -0,0 +1,84 @@
+# Polywrap Python Client - https://polywrap.io
+# SHA3 Wrapper Schema - https://wrappers.io/v/ipfs/QmbYw6XfEmNdR3Uoa7u2U1WRqJEXbseiSoBNBt3yPFnKvi
+
+from pathlib import Path
+from polywrap_client import PolywrapClient
+from polywrap_core import Uri, InvokerOptions
+
+client = PolywrapClient()
+uri = Uri(f'fs/{Path(__file__).parent.joinpath("cases", "simple-invoke", "wrapperSHA3.wasm").absolute()}')
+args = {"message": "hello polywrap!"}
+
+async def test_invoke_sha3_512():
+    options = InvokerOptions(uri=uri, method="sha3_512", args=args, encode_result=False)
+    result = await client.invoke(options)
+    print(result)
+    assert result.result == False
```

Comment:
Why result is False in every case?

Reply:
Updated the tests so it includes the correct assertions for almost every unit test. They still cant pass as green due to the `wasmtime` error mentioned above in the PR. 

also missing the right assertion for `test_invoke_buffer_keccak_256`